<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_category_widget</class>
 <widget class="QWidget" name="add_category_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Category</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Files/Icons/plus-16.ico</normaloff>../Files/Icons/plus-16.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="category_name_input">
     <property name="placeholderText">
      <string>Category name...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>add_category_widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>add_category_widget</receiver>
   <slot>addbuttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addbuttonClicked()</slot>
 </slots>
</ui>
